### PR TITLE
Create helm chart for blackbox exporter

### PR DESCRIPTION
**Pull request by gianrubio**

### CODE REVIEW

**Comment by bhack** [comment]
Any news on this?

File: stable/prometheus-blackbox-exporter/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Prometheus Blackbox Exporter
+
+Prometheus exporter for blackbox testing
+
+Learn more: [https://github.com/prometheus/blackbox_exporter](https://github.com/prometheus/blackbox_exporter])
```
Comment (bhack) [inline stable/prometheus-blackbox-exporter/README.md:5]: Too late.. but there is an extra `]` that borkes the link

File: stable/prometheus-blackbox-exporter/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+$ helm install --name my-release \
+    --set key_1=value_1,key_2=value_2 \
+    stable/blackbox-exporter
+```
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```bash
+# example for staging
+$ helm install --name my-release -f values.yaml stable/blackbox-exporter
```
Comment (bhack) [inline stable/prometheus-blackbox-exporter/README.md:82]: The name is `stable/prometheus-blackbox-exporter`

**Comment by bhack** [comment]
It was merged too fast..

**Comment by gianrubio** [comment]
@bhack would you mind to shoot a PR with these fixes?

**Comment by bhack** [comment]
@gianrubio It is not only these I think that you need to check also the configmap template name cause it is not matching after depolyed.
So I suggest you to do a general review of this PR and open a new one when you are sure that it is really working.

Thanks

**Comment by desaintmartin** [comment]
Are there still issues after the new PR?

**Pull request merged**